index.html

```diff
@@ -0,0 +1,223 @@
+      <div class="text">
+        <h2 class="Tonic">Tonic</h2>
+        <div class="list">
+          <ul>
+            <li class="CANOPY">CANAPY</li>
+            <li>
+              <a href=""><img src="images/Counter.png" alt="" /></a>
+            </li>
+            <li>Back End Dev</li>
+            <li>
+              <a href=""><img src="images/Counter.png" alt="" /></a>
+            </li>
+            <li>2022</li>
+          </ul>
+        </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts
+          or sign-ups required
+        </p>
+        <div class="button-list">
+          <ul>
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javaScript</a></li>
+          </ul>
+        </div>
+        <div>
+          <button class="btn" type="button">See project</button>
+        </div>
+      </div>
+    </section>
+  </section>
+  <section class="about">
+    <div>
+      <div class="text">
+        <h2>About<br> Myself</h2>
+        <p>
+          Hello I’m a data Scientist and software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <footer class="footer">
+          <h3>LET'S CONNECT</h3>
+          <div class="contact">
+            <ul>
+              <li class="icon"><a href="#" target="_blank"><img src="./images/twitter.png" alt="linked link"></a></li>
+              <li class="icon"><a href="#" target="_blank"><img src="./images/Linkedin_icon.png" alt="github link"></a></li>
+              <li class="icon"><a href="#" target="_blank"><img src="./images/M.png" alt="linked link"></a></li>
+              <li class="icon"><a href="#" target="_blank"><img src="./images/github.png" alt="github link"></a></li>
+              <li class="icon"><a href="#" target="_blank"><img src="./images/doits.png" alt="Angel list link"></a></li>
+            </ul>
+          </div>
+        </footer>
+        <div>
+          <button class="btn" type="button">Get my resume</button>
+        </div>
+      </div>
+      <div>
+        <ul>
+          <li class="options">
+            <h3>Languages</h3>
+            <div><img class="hide" src="images/fleche_dessous.png" alt="selected"></div>
+          </li>
+          <li>
+            <div>
+              <ul>
+                <li class="pro">
+                  <img src="images/java.png" alt="javascript image">
+                  <p class="">JavaSript</p>
+                </li>
+                <li class="pro">
+                  <img src="images/html.png" alt="javascript image">
+                  <p class="">html</p>
+                </li>
+                <li class="pro">
+                  <img src="images/css.png" alt="javascript image">
+                  <p class="">css</p>
+                </li>
```

Comment:
- [ ] [OPTIONAL] Your design here looks great but It is also important to pay attention to using semantic tags as much as possible. That is why I would suggest you use the concept of Nested lists here in this section. You can refer to this [document](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/). It will give a thorough understanding of what you can improve and by doing this, your HTML page will be more semantic 😃 👍 .